unions-intersections/Exercise3/solution/Solution.ts

```diff
@@ -0,0 +1,57 @@
+interface Customer {
+  id: string;
+  fullName: string;
+  lifeTimeValue: number;
+}
+
+interface Employee {
+  salary: number;
+  performance: "great" | "good" | "meh" | "needs training";
+}
+
+interface BoardMember {
+  ownership: number;
+}
+
+type Person = (Customer & Employee) | (Customer & BoardMember) | Customer; // Fixed
+// or
+// type Person = Customer & (Employee | BoardMember | {})
```

Comment:
I wonder if this is actually the better solution and should be the one we encourage.🤷‍♂️